cpp/src/pca/pca_mg.cu

```diff
@@ -137,10 +137,10 @@ void fit_impl(raft::handle_t& handle,
     const auto& comm        = h.get_comms();
 
     // Center the data
-    Matrix::Data<T> mu_data{mu, size_t(prms.n_cols)};
+    Matrix::Data<T> mu_data{mu, prms.n_cols};
     Stats::opg::mean(handle, mu_data, input_data, input_desc, streams, n_streams);
     Stats::opg::mean_center(input_data, input_desc, mu_data, comm, streams, n_streams);
-    for (int i = 0; i < n_streams; i++) {
+    for (std::size_t i = 0; i < n_streams; i++) {
```

Comment:
Tiny nitpick- this one could stay an int or be unsigned. Using 32-bit might be more appropriate since n_streams should be 32-bit.